stress_test/controller.py

```diff
@@ -78,11 +87,15 @@ def init_ssh(self):
         logging.info(
             '[open_ssh_connection] Initiating SSH session with {0} node.'.
             format(self.name, self.ip))
-        self._ssh_conn = util.netutil.ssh_connect_or_return2(self.ip,
-                                      int(self.ssh_port),
-                                      self.ssh_user,
-                                      self.ssh_pass,
-                                      10)
+#        self._ssh_conn = util.netutil.ssh_connect_or_return2(self.ip,
```

Comment:
Use this version. 

Using named tuples to attack large argument lists (which by the way is not an issue for `ssh_connect_or_return`) is a practice we should adopt in extreme cases. 

For example, see paramiko's `connect` method here how many arguments it takes:  http://stackoverflow.com/questions/11551043/paramiko-ssh-connect-what-arguments-to-send